luigi/task.py

```diff
@@ -276,7 +276,7 @@ def trigger_event(self, event, *args, **kwargs):
                 except KeyboardInterrupt:
                     return
                 except BaseException:
-                    logger.exception("Error in event callback for %r", event)
+                    logger.exception("Error in event callback (%s) for %r", callback, event)
```

Comment:
Always put changes like this in separate PRs please

Reply:
ahh, sry. i believe it was in another pr somewhere, thought this got merged into master... but maybe not...